rust/rsmgp-example/src/lib.rs

```diff
@@ -0,0 +1,104 @@
+use backtrace::Backtrace;
+use c_str_macro::c_str;
+use rsmgp_sys::memgraph::*;
+use rsmgp_sys::mgp::*;
+use rsmgp_sys::property::*;
+use rsmgp_sys::result::*;
+use rsmgp_sys::rsmgp::*;
+use rsmgp_sys::value::*;
+use rsmgp_sys::{close_module, define_procedure, define_type, init_module};
+use std::ffi::CString;
+use std::os::raw::c_int;
+use std::panic;
+
+init_module!(|memgraph: &Memgraph| -> Result<(), MgpError> {
+    memgraph.add_read_procedure(
```

Comment:
Where are input parameters defined?

Reply:
Yea, this is a good point, but that's the `macro` where any [Expression](https://doc.rust-lang.org/reference/expressions.html) can be passed, but if it's wrong, it's going to fail because of the code "inside" `init_module!`. Basically, `|memgraph: &Memgraph| -> Result<(), MgpError>` signature is required, but I don't know how to enforce that because Rust macros just have these "fragment-specifiers" https://doc.rust-lang.org/reference/macros-by-example.html Probably it's possible somehow, but I just don't think it's worth it at this point... But yea, if there is a simple syntax to do so, I can add :smile: